### DISCUSSION TITLE
Github Classroom Reuse Assignment Feature->JSON Error. Cannot copy

### DISCUSSION THREAD

**Discussion by tturin1017** [community/community#138211] [GitHub Education]
### Hi everyone,

Hi,

I wasn't sure where to post this issue under the discussion topics. Last year I didnt' have this issue copying assignments to different classroom but I am getting a JSON error when attempting to reuse an assignment. 
Unexpected token '<', "<!doctype "... is not valid JSON
<img width="406" alt="Screenshot 2024-09-09 133815" src="https://github.com/user-attachments/assets/761e5d3f-0996-40ee-8b0f-b511ec81f87b">

Any suggestions? Thanks!

**Comment by ExploreLearnMake** [comment]
I am getting the same error message. Trying to bump this to see if someone can help find a solution.

**Comment by peihwee** [comment]
I also getting the same message.

**Comment by wheat32** [comment]
I also get the same error. Would like to see this resolved.

**Comment by AdamFiser** [comment]
Same bug for me

**Comment by BSU-devharsh** [comment]
Unexpected token '<', "<!doctype "... is not valid JSON
